docs/examples/search-session.md

```diff
@@ -61,46 +61,34 @@ Epochs
 Now that we've got those, here's a small dump of some example data you
 could imagine comming from your app's events infrastructure.
 
+Let's write a function that will yield one of these events at a time
+into our dataflow.
+
 ```python
 IMAGINE_THESE_EVENTS_STREAM_FROM_CLIENTS = [
-    # epoch, event
-    (0, AppOpen(user=1)),
-    (1, Search(user=1, query="dogs")),
+    AppOpen(user=1),
+    Search(user=1, query="dogs"),
     # Eliding named args...
-    (2, Results(1, ["fido", "rover", "buddy"])),
-    (3, ClickResult(1, "rover")),
-    (4, Search(1, "cats")),
-    (5, Results(1, ["fluffy", "burrito", "kathy"])),
-    (6, ClickResult(1, "fluffy")),
-    (7, AppOpen(2)),
-    (8, ClickResult(1, "kathy")),
-    (9, Search(2, "fruit")),
-    (10, AppClose(1)),
-    (11, AppClose(2)),
+    Results(1, ["fido", "rover", "buddy"]),
+    ClickResult(1, "rover"),
+    Search(1, "cats"),
+    Results(1, ["fluffy", "burrito", "kathy"]),
+    ClickResult(1, "fluffy"),
+    AppOpen(2),
+    ClickResult(1, "kathy"),
+    Search(2, "fruit"),
+    AppClose(1),
+    AppClose(2),
 ]
-```
-
-Let's breakdown some of the details here.
 
-One of Bytewax's cool powers is giving you a high level control over
-the concept of time in your execution via attaching a kind of
-timestamp called an **epoch** to each **item** of input data.
-
-[You can read more about the details of epochs in our
-documentation](/getting-started/epochs/), but the most important facet of them for this
-example is that they are the _only_ way you can set up a sense of
-order in your data.
-
-Since we are going to use the order in which events happen to divvy
-them up into user sessions, we need to take advantage of epochs at
-some level. In this basic example, we're going to label each input
-event with a monotonically increasing epoch. [There are some some
-scaling downsides to this approach](/getting-started/epochs/), but we will be
-using it here anyway to demonstrate the basics of epochs.
+def input_builder(worker_index, worker_count, resume_state):
+    state = resume_state or {}
```

Comment:
@whoahbot if you were actually using a state passed in, would this work?

Reply:
I prefer the `None` scenario so we don't have to think hard about the shape of a `state` that we're ignoring, though in general this concern points more to @davidselassie's desire to move away from this shape in general.

I'll update it to `None` for now but we can revisit when you're back.